src/js/review/ArrayField.jsx

```diff
@@ -187,7 +189,17 @@ class ArrayField extends React.Component {
             <h5 className="form-review-panel-page-header">{title}</h5>
             {itemsNeeded && <span className="schemaform-review-array-warning-icon"/>}
             {!itemCountLocked &&
-              <button type="button" className="edit-btn primary-outline" onClick={() => this.handleAdd()}>
+              <button type="button"
+                disabled={addAnotherDisabled}
+                className={classNames(
+                  'edit-btn',
+                  'primary-outline',
+                  {
+                    'edit-btn-disabled': addAnotherDisabled
```

Comment:
I tried using `usa-button-disabled` but there's a good bit of cruft on the review page that would require some refactoring. most of the buttons on this page have their own styles that deviate.

Reply:
Do we need the extra class, or could we style against `.edit-btn:disabled`?

Reply:
Our comments crossed, I'll wait until this is resolved before I merge. Since you've brought this up, is there some visual indication to the user that they've reached the max, other than the disabled button?

Reply:
Since this is blocking work I was going to suggest we handle that in a separate issue. I think we should display the max- but I'd like to get [this](https://github.com/usds/us-forms-system/pull/333#issuecomment-459021124) merged in too so we can handle it together. Seems like there's more space to do it at the bottom of the array list anyway.

Reply:
Removed the additional class and updated the formation [behavior](url
https://github.com/department-of-veterans-affairs/design-system/pull/233) to select based on the `disabled` attribute.

Reply:
It's important to give people some understanding of why they hit the limit, but it could be as simple as putting a sentence in the form content somewhere that says "Enter your most recent five" or words like that. If that's the way it's handled it may not require any code changes here.